src/pages/api/docker/DockerSingleton.tsx

```diff
@@ -0,0 +1,16 @@
+import Docker from 'dockerode';
```

Comment:
I thought this would create an API route to /api/docker/DockerSingleton but I belive since no "handler" function is exposed then it won't. Can you verify if this is true ? @Skeeww

Reply:
Exactly it won't be exposed, it purpose is to be called only with API's routes